simdata/simulate_data_l1.ipynb

```diff
@@ -0,0 +1,1061 @@
DQgFQqBRGVm2/16tXYvHkzevXqhQULFqBdu3aIiYmBjY0NTp06JZwgISEh8PDwkNuvsrRu3VroDTY0NAQR4cqVKwCK51zIzMx8n6IEANjY2GDv3r1C2fn6+go9+crs27cPPj4+sLGxwZw5c2BjY4NHjx4prPeXX34BEUEsFuPgwYPCvAvKKCu3f5KjoyMuXryI9PR0EBEOHDggvC/cs2dPoc68ePECV69ehYODAzp37oy3b98Ks8CHhobCzMwMderUqXB7Z86cgVgsRmFhIY4cOVKpd+Vmz56NnJwcuc4KoLjMIyIikJCQAADYv3+/MJIFKP6VG7FYjGbNmlW6PNinT9n52qNHD5w9exaZmZmQyWRys9PXr19feAL/8uVL/PnnnwBQpXbXysoK169fF2bxvnLlCgYMGICCggK5fNbW1rh+/bowQqvk6REAXLt2DR4eHhg4cCAaNGiAGzduQCqVApBvM5Xly8jIgL29PerVqwdPT0/MmDEDDx48QK1atWBmZoaff/4ZQPETypEjR+LChQsK6y6tdevWSE5OFv5ta2uLo0ePorCwEIWFhTh9+nRVDo0cMzMzJCYmCk+dHj9+DCcnJ7x8+VLpMkVFRXB0dER+fj5GjhwJPz8/REVFQSwWC3msrKxw7do1Ie6SOYnKGqlSQlm5/ZM0NDTQo0cPHDx4EADw5MkTxMXFoVu3buW2dT179kRoaCiKioqQlZWFU6dOVWrOh4yMDPz+++8Ait9519TUlJtPRRViT0lJQZs2bd4rJlb9evXqhTVr1sDa2hpt27ZFTk4OTpw4gT59+sjl69q1K2JjY/HkyRMAkPsFu/KuI0uzs7PD2bNnkZWVBZlMJteJcO3aNUyZMkUYTRYREaG0XS0r35MnT+Di4oK2bdti4sSJ8PT0xIMHD9C6dWvo6OgI23r+/DlcXFyE75KqtKuHDh2CTCZDZmam0C6/j8p+D5VWmfbP1tYWp0+fFn7BIjQ0VGGukHcpK7d/UpMmTdCiRQvhu+rq1atQU1ODgYHBe13/VuTJkydCPT5w4ADMzc0rdW38T8Re0bU7EeHZs2do3br1e8VbXXiEhYro27cv3N3dsWHDBvTo0QP9+vWDlpaWMHwuMTFRode1LH5+fvDx8UG/fv3QpEkToafQ2toahw4dQt++fSESiWBpaQk9PT0kJiaiZcuWMDU1hbOzM37++Wc0bty4zHweHh7w9vYWhgobGhrC2dkZWlpaGD9+PMaOHQuRSIRatWph48aNEIlEMDMzw6ZNmzB16lRs3LhRLlYnJyecP38eQ4YMgaamJjZt2oRFixZh7dq16Nixo9wrHFU1efJkBAQEYNCgQZBKpejYsaPws3zKDBw4EH/++Sf69++PGjVqoGnTpnB3dxcaJaC459ff3x+urq6QSCSwtbVV+PnQdykrN0D5pJsfWocOHTBlyhR4eHhAIpGgc+fOwhDradOmYdGiRXB2doZUKsWcOXOEYYwbN27EkiVLkJ+fj3r16iEgIABA8c3ehAkTsG3btjJHXOjo6GDUqFHIysqCk5MThgwZUm589+7dw7lz59CqVSuMHDlS+Hz27NmwtbXF8uXLMX36dEgkErRo0UKIAyi++OjRo0elzg/276HsfK1bty7GjBmDUaNGQVtbW3iVAyieMHL27NlwcnKCvr4+vvzySwDFHaaVbXdLJmybNWsWiAgaGhrYsmWL3EisknXOmTMHHh4e0NXVlZscccqUKVi5ciU2bNgATU1NfPHFF0LnrZ2dHVasWFFuPj09PfznP/+Bp6cndHR0oK6uDn9/fwDFHaRLly6Fq6srxGIxXFxcMGDAAADFw7fnzJmDRYsWwcbGRojHwMAA2traiIuLQ9u2bTFixAgkJSXBxcWlwovViujp6SEwMBArV65EYWEhiAgrV66Evr6+0GH0Lg0NDcyfPx+zZ8+GhoYGRCIRfvjhB7lj0a5dO/j5+WHq1KmQSqXQ0dFBcHBwuU89yys3ZZNufgx+fn5YuHAhXFxcIBKJsHLlSiFuZW3dyJEjkZSUBDc3N0gkEgwfPhyWlpYAoHRiO6B41N+xY8ewevVq6OjoYNOmTX/rydqHjh0ovkAfPXr0e8fEqlfv3r2xY8cO4eFN9+7dERUVJTcBJVB8/q1evRqzZ8+GpqYmunbtKqTZ2NgovY4szd7eHlFRURgyZAjq1KmDDh06CBOaz5w5E1OmTEHNmjVRq1YtdO3aVWhXHR0dsXbtWkgkEqX5hg0bhn79+mHIkCGoWbMmdHR0sHDhQmhpaWHz5s1YtmwZtm/fjqKiInz77bfCpIYl1+9BQUFynYHdu3fHggULkJWVhTp16mDatGnw8/NDv379oKen994dh0Dlv4feLX9l7V8Ja2treHp6wsPDAzKZDHp6eti6dWu5I6A6dOhQZrkB5bdNH9ratWvh6+uLLVu2QEtLCxs2bICamtp7X/8uWLAAxsbGctekJT777DOsX78eqamp0NPTw8qVK1UqdmXX7kDxT6y3aNHik3vIJ6LKvmvA2AcmlUoxePBgpTe9TLV5e3sLv7zy7uelf4nhYxszZgzmz59fqWF87P+fs2fPYu/evcJ7zqxsJ06cwJ07d7Bo0aLqDoVVUUJCAg4dOoTZs2fLfZ6SkgJXV1fcu3evmiKrWFJSEmbPno0DBw6UOQcGY5+y4OBgqKurK8zBw1Tf9evXkZSUpNBhUfqXCz9F3t7e6Nu3L3r06FHdoVQJvxLCqo26ujqWLl1a4asaTPXk5+fDyspKobPin3b+/Hl06dKFOysY+5tcXV2RmZlZ5uSaTLXFx8eXOaHxp2D9+vXw9/fnzgr2rzR27FjcvHkTr169qu5QWBW9fftWmLD43yIyMhIikeiT66wAeIQFY4wxxhhjjDHGVBCPsGCMMcYYY4wxxpjK+SQn3ZTJZMjNzVX4STXGGPs3ICJIJBLo6uqq1M/9cdvLGPs347aXMcb+eRW1vZ9kh0Vubi6io6OrOwzGGPuoDA
```

Comment:
Nice! Weird that the sklearn logistic regression is consistently doing worse than random chance on the tall matrices though